Bottom: c390dfee9b93bf7bc5ac5e78cdcb271652ddf864
Top:    1b6b60ecdca3a324615d1e5a20450d248a682892
Author: scott snyder <sss@karma>
Date:   2020-02-12 04:05:55 -0500

Fix compilation warnings.


---

diff --git a/src/DDConeMeasLayer.cc b/src/DDConeMeasLayer.cc
index b0065d2..7d80e8f 100644
--- a/src/DDConeMeasLayer.cc
+++ b/src/DDConeMeasLayer.cc
@@ -104,16 +104,16 @@ Bool_t DDConeMeasLayer::IsOnSurface(const TVector3 &xx) const {
   Double_t r   = xxc.Perp();
   Double_t z   = xxc.Z();
   Double_t s   = (r - GetTanA()*z) * (r + GetTanA()*z);
-  const Double_t kTol = 1.e-8;
+  const Double_t tol = 1.e-8;
 
 #if 0 
   std::cout << this->TVMeasLayer::GetName() << ":" << this->GetIndex() << ":" 
 	    << "s=" << s << " xx=(" << xx.X() << "," << xx.Y() << "," << xx.Z() << ")" 
-	    << "bool=" << (TMath::Abs(s) < kTol && ((xx.Z()-_Z1)*(xx.Z()-_Z2) <= 0.)) 
+	    << "bool=" << (TMath::Abs(s) < tol && ((xx.Z()-_Z1)*(xx.Z()-_Z2) <= 0.)) 
 	    << "_Z1=" << _Z1 << " _Z2=" << _Z2 << std::endl;
 #endif
 
-  return (TMath::Abs(s) < kTol && ((xx.Z()-_Z1)*(xx.Z()-_Z2) <= 0.));
+  return (TMath::Abs(s) < tol && ((xx.Z()-_Z1)*(xx.Z()-_Z2) <= 0.));
 } 
 
 Int_t DDConeMeasLayer::CalcXingPointWith(const TVTrack  &hel,
